src/test/java/uk/gov/digital/ho/hocs/casework/migration/api/MigrationCaseDataServiceTest.java

```diff
@@ -125,7 +125,6 @@ public void shouldCreateOpenMigrationCase() throws ApplicationExceptions.EntityC
             originalReceivedDate, null, null, originalCreatedDate, migratedReference);
 
         // then
-        assertThat(caseData.isCompleted()).isEqualTo(false);
```

Comment:
This and the other modified tests in this file should be replaced with a check that date completed has been set.